decidim-core/db/migrate/20210210114657_add_external_domain_whitelist_to_organization.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddExternalDomainWhitelistToOrganization < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_organizations, :external_domain_whitelist, :string, array: true, default: []
```

Comment:
Let's add `decidim.org` and `github.com` to all existing organizations after the column is added in this migration.

Also, let's change the logic for creating a new organization which should also do the same (add these two domains to the whitelist by default).

The reason for this is that these links are in the footer of every Decidim instance by default.